backend/src/services/token/token.service.ts

```diff
@@ -0,0 +1,19 @@
+import * as jose from 'jose';
+import { ENV } from '~/common/enums/enums';
+
+class Token {
+  #secretkey = new TextEncoder().encode(ENV.JWT.SECRET);
+
+  create(data: jose.JWTPayload): Promise<string> {
```

Comment:
```suggestion
type TokenPayload = {
  userId: number;
};

  create(data: TokenPayload): Promise<string> {
```
create a token in the `backend/src/common/types/token/token-payload.type.ts`